### PR TITLE
Update endpoint-dlp-learn-about.md

### PR DESCRIPTION
**Pull request by msbemba**
Minumum windows 10 build number to be complaint is Version 10.0.17134.1. Added that to the document.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2471

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/endpoint-dlp-learn-about.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ description: "Microsoft 365 Endpoint data loss prevention extends monitoring of
...
 
-**Endpoint data loss prevention** (Endpoint DLP) extends the activity monitoring and protection capabilities of DLP to sensitive items that are on Windows 10 devices. Once devices are onboarded into the Microsoft 365 compliance solutions, the information about what users are doing with sensitive items is made visible in [activity explorer](data-classification-activity-explorer.md) and you can enforce protective actions on those items via [DLP policies](create-test-tune-dlp-policy.md).
+**Endpoint data loss prevention** (Endpoint DLP) extends the activity monitoring and protection capabilities of DLP to sensitive items that are on Windows 10 devices. This works on all windows 10 devices which are on or above [Version 10.0.17134.1]. Once devices are onboarded into the Microsoft 365 compliance solutions, the information about what users are doing with sensitive items is made visible in [activity explorer](data-classification-activity-explorer.md) and you can enforce protective actions on those items via [DLP policies](create-test-tune-dlp-policy.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-learn-about.md]:
```suggestion
**Endpoint data loss prevention** (Endpoint DLP) extends the activity monitoring and protection capabilities of DLP to sensitive items that are on Windows 10 devices. This works on all Windows 10 devices which are on or above build 10.0.17134.1. Once devices are onboarded into the Microsoft 365 compliance solutions, the information about what users are doing with sensitive items is made visible in [activity explorer](data-classification-activity-explorer.md) and you can enforce protective actions on those items via [DLP policies](create-test-tune-dlp-policy.md).
```

Comment (JohanFreelancer9) [reply microsoft-365/compliance/endpoint-dlp-learn-about.md]: Please correct the added copy as indicated.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
wrong place for this,, it is addressed in the Getting started topic

**Pull request closed without merging**